### ISSUE TITLE
1.1.338: False positive passing `type` in for `Callable[..., Any]`

### ISSUE THREAD

**Issue by rsokl** [microsoft/pyright#6600]
```python
from typing import Callable, Any

def f(x: Callable[..., Any]): ...

def g(x: type):
    # pyright 1.1.337: OK
    # pyright 1.1.338: Argument of type "type" cannot be assigned to parameter "x" of type "(...) -> Any" in function "f"
    f(x)  
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.339, which I just published. It will also be included in a future release of pylance.
